netbox/netbox_inventory.py

```diff
@@ -132,53 +182,84 @@ def fetch_inventory(cli_opts):
     if cli_opts.region:
         params["region"] = cli_opts.region
 
-    res = netbox.get("/api/dcim/devices/", params=params)
+    res = NETBOX_SESSION.get("/api/dcim/devices/", params=params)
```

Comment:
It's recommended to handle exceptions for network requests to improve the robustness of your network interactions. [important]